colony/shell.py

```diff
@@ -5,10 +5,10 @@
 Options:
   -h --help             Show this screen.
   --version             Show current version
-  --space=<space>       Colony Space name
-  --token=<token>       Specify token generated by Colony
-  --profile=<profile>   Profile indicates a section in config file.
-                        If set neither --token or --space must not be specified.
+  --space=<space>       Use a specific Colony Space, this will override any default set in the Colony config file

```

Comment:
we need to have common way to for this messages.
For space you are saying "Colony config file". For token and profile you are saying ".colony\\config file".
But maybe its better to say in all 3 places:
"%USERPROFILE%\\.colony\\config file for windows or "~\.colony\\config file for linux"

Reply:
Might be too verbose. I saw git in their documentation use 'global config file' or 'user config file' (without specifying where it is stored). I didn't fing a place when I'm mentioning ~.colony\config any more

Reply:
I agree, no need to talk about this level of details here. So can you change ".colony\\config" in --token and --profile to just Colony config file?

Reply:
it already says that no?
Where do you see .colony\config?